SimplenoteTests/NSStringSimplenoteTests.swift

```diff
@@ -56,4 +56,22 @@ class NSStringSimplenoteTests: XCTestCase {
         XCTAssertEqual(hashA, hashC)
         XCTAssertEqual(hashB, hashC)
     }
+
+    /// Verifies that `isValidEmailAddress` returns  *true* when the receiver contains an email address with a non-standard TLD
+    ///
+    func testIsValidEmailAddressReturnsTrueWhenStringContainsNewTLDs() {
```

Comment:
I'm wondering if these tests would be better as part of `NSPredicateEmailTests` since that's the underlying logic being tested. The String extension is just for convenience.  
Just a thought.  No worries if you prefer it this way.